### ISSUE TITLE
"show all" -> browse output empty - not even a empty result

### ISSUE THREAD

**Issue by M-Mommsen** [phpmyadmin/phpmyadmin#17481]
When I click the "show all" checkbox, the browse tab is completely empty (any table that has rows, even only a single row). Even after navigating this problem persists for this table. Current workaround is to use search tab with no filters. But using the browse tab for this specific table stays broken!
Logoff and login resets the "show all", and browse tab is working again.

Using git master, and sample config. Only ran composer and yarn update (all as www user).
PHP version 8.0.17
master / 5.3.0-dev / Git revision: [f8c04a1 ]
Tried different theme.

![image](https://user-images.githubusercontent.com/7465071/163128112-6a477e37-c76d-4f53-82b6-222d24498f93.png)

**Comment by williamdes** [comment]
Hi @M-Mommsen 
Can you try the latest 5.2 version in development [(phpMyAdmin 5.2+snapshot)](https://www.phpmyadmin.net/downloads/#snapshot_5.2+snapshot) ?

Because 5.3 is a dev version for now.

Let me know if this is still an issue on 5.2

**Comment by M-Mommsen** [comment]
I checked the stable branch. A litte bit different result. 
ShowAll is loading forever. I tried debugging it, and came over a json parse error.
The json is no json at all - it's a html page containing a "customize browse mode" page, with a bunch of html configuration elements.
I switched to php7.4 (from php8.0) and the problem is gone.
I can't currently switch back to php8 to give more information.
If nobody else is experiencing this issue just close it. For now I will just stick to php7.4
